Files/Dialogs/DecompressArchiveDialog.xaml

```diff
@@ -0,0 +1,50 @@
+﻿<ContentDialog
+    x:Class="Files.Dialogs.DecompressArchiveDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:helpers="using:Files.Helpers"
+    xmlns:local="using:Files.Dialogs"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    Title="Select a destination and extract files"
+    CornerRadius="{StaticResource OverlayCornerRadius}"
+    DefaultButton="Primary"
+    PrimaryButtonText="Extract"
+    RequestedTheme="{x:Bind helpers:ThemeHelper.RootTheme}"
+    SecondaryButtonText="Cancel"
+    Style="{StaticResource DefaultContentDialogStyle}"
+    mc:Ignorable="d">
+
+    <Grid
+        MinWidth="400"
+        Margin="0,16,0,0"
+        ColumnSpacing="6"
+        RowSpacing="3">
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
+
+        <TextBox
+            HorizontalAlignment="Stretch"
+            VerticalAlignment="Center"
+            IsReadOnly="True"
+            Text="{x:Bind ViewModel.DestinationFolderPath, Mode=OneWay}" />
```

Comment:
Please add strings and placeholder text in case the user clears the text in the textbox.

Reply:
The text box is read only. There's always a placeholder path set in the view model.

Reply:
Is it localized?

Reply:
huh? Only the destination path is displayed here...